app/controllers/trainees/confirm_details_controller.rb

```diff
@@ -48,11 +48,21 @@ def form_klass
     end
 
     def trainee_section_key
-      request.path.split("/").intersection(trainee_paths).first&.underscore
+      @trainee_section_key ||= request.path.split("/").intersection(trainee_paths).first&.underscore
     end
 
     def confirm_section_title
-      trainee_section_key.gsub(/_/, " ").gsub(/id/, "ID").gsub(/training\sdetails/, "trainee start date and ID")
+      @confirm_section_title ||=
+        begin
+          case trainee_section_key
+          when "training_details"
+            "trainee start date and ID"
+          when "lead_school"
+            "training details"
```

Comment:
Is this the title of the confirmation component? If so, I think it's now been changed to 'Schools' in the prototype:

![Uploading Screenshot 2021-04-29 at 10.28.32.png…]()

Reply:
@AbigailMcP this is the title of the page "Confirm training details". I'll double check the prototype to make sure the page matches.

Reply:
Ah! I see - yeah  it's "Confirm schools":

<img width="1031" alt="Screenshot 2021-04-29 at 10 36 59" src="https://user-images.githubusercontent.com/18436946/116531263-e26a0a00-a8d6-11eb-99ea-7b0b52a502be.png">